src/ManageCourses.Domain/Models/Course.cs

```diff
@@ -61,7 +61,7 @@ public Course()
         public IEnumerable<Site> Sites { get => CourseSites != null && CourseSites.Any() ? CourseSites.Select(x => x.Site) : null; }
 
         [NotMapped]
-        public EnumStatus? EnrichmentWorkflowStatus { get; set; }
+        public WorkflowStatus? EnrichmentWorkflowStatus { get; set; }
```

Comment:
Can you slap a comment, along the lines that if its null then it was not set

ie the DataService.xxx is the only method that should be setting...

Just in case we need to revisit/build on this

Reply:
Done